src/components/place-card-favorite/place-card-favorite.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import PropTypes from "prop-types";
+import PlaceCard from "../place-card/place-card";
+
+const PlaceCardNear = (props) => {
+  const {className = ``} = props;
+  const restProps = Object.assign({}, props);
+  delete restProps.className;
+
+  return (
+    <PlaceCard className={`favorites__card ${className}`} {...restProps}
```

Comment:
```suggestion
  delete restProps.className;

  return (
    <PlaceCard {...restProps} className={`favorites__card ${className}`}
```

если сделать так, то className в пропсах перезатрется, и не надо будет его удалять вручную